internal/cfgplugins/interface.go

```diff
@@ -899,10 +901,9 @@ func NewAggregateInterface(t *testing.T, dut *ondatra.DUTDevice, b *gnmi.SetBatc
 		lacp.LacpMode = *l.LacpParams.Activity
 		lacp.Interval = *l.LacpParams.Period
 		lacpPath := gnmi.OC().Lacp().Interface(aggID)
-		gnmi.BatchReplace(b, lacpPath.Config(), lacp)
+		gnmi.BatchUpdate(b, lacpPath.Config(), lacp)
```

Comment:
I don't understand why this cannot be part of the BatchReplace?     If that doesn't work, it's probably a bug in the DUT?